src/features/post/new/PostEditorRoot.tsx

```diff
@@ -419,6 +421,26 @@ export default function PostEditorRoot({
                   value={url}
                   onIonInput={(e) => setUrl(e.detail.value ?? "")}
                 />
+                {!!jwt && !!url && isValidUrl(url) && (
+                  <IonButton
+                    onClick={(e) => {
+                      e.preventDefault();
+                      function toast() {
+                        presentToast({
+                          message: "Unable to fetch title",
+                          color: "danger",
+                        });
+                      }
+                      getSiteMetadata(url, instanceUrl, jwt)
+                        .then((data) =>
+                          data.title ? setTitle(data.title) : toast(),
+                        )
+                        .catch(toast);
+                    }}
+                  >
+                    FETCH TITLE
```

Comment:
Can we do something shorter and more subtle, maybe remove the bg too? Maybe there's an appropriate icon we can use (https://ionic.io/ionicons)

Reply:
I was initially experimenting with some of the icons but wasn't sure which one to go with, so I just went with a button for now. open to suggestions for the icon for this button :v: